Example/UnioSample.xcodeproj/project.pbxproj

```diff
@@ -124,6 +127,9 @@
 		9D2E26EC2240D96000C9EDF7 /* Frameworks */ = {
 			isa = PBXGroup;
 			children = (
+				C51136AF27B38B8800E8F26C /* RxCocoa.xcframework */,
+				C51136AC27B38B7400E8F26C /* RxRelay.xcframework */,
+				C51136A527B38B1100E8F26C /* RxSwift.xcframework */,
 				9D9EEC0E22816B1700DF5D97 /* RxRelay.framework */,
 				9D2E26F02240D98300C9EDF7 /* RxCocoa.framework */,
 				9D2E26EF2240D98300C9EDF7 /* RxSwift.framework */,
```

Comment:
Could you delete framework references?

Reply:
Fixed it https://github.com/cats-oss/Unio/pull/25/commits/3c37dc910afac72d6f7371af6afa1736145307b6